docs/BestPractices.md

```diff
@@ -23,6 +23,13 @@ FROM node:4.1.2-onbuild
 RUN groupadd -r node && useradd -r -g node node
 ```
 
+When using the `alpine` variant, add the user like so:
+
+```Dockerfile
+FROM node:7.3.0-alpine
+RUN addgroup -S node && adduser -S -g node node 
+```
```

Comment:
When did this get changed? I see my latest build was from 20 days ago and the complete dockerfile is the following:

```Dockerfile
FROM node:7.2.0-alpine
RUN addgroup -S nodeapp && adduser -S -g nodeapp nodeapp 
WORKDIR /app
COPY . .
RUN npm install && npm run build && npm run test
EXPOSE 3000
USER nodeapp
CMD ["node", "src/server.js"]
```